src/main/resources/webapp/js/controllers.js

```diff
@@ -1191,6 +1191,29 @@
         $scope.newForm = function() {
             $scope.form = {};
             $scope.form.dto = {};
+            $scope.form.dto.minActualDate = null;
```

Comment:
this should be calculated in the directive and in ignoreDateLimitation watch method

Reply:
I agree that it should be calculated in directive but I am thinking about ignoreDateLimitation watch method. I think that it shouldn't be there because no matter of ignoreDateLimitation value actual date min and max should be always the same, all dates in past including today.